Include <gcrypt.h> in the libgcrypt version prope, so that the
compilation does not file if the compiler does not support implicit
function declaratiosn.

Submitted upstream: <https://rt.cpan.org/Public/Bug/Display.html?id=146086>

diff --git a/Makefile.PL b/Makefile.PL
index 86de08279df14c8f..3821c1cd46524ddc 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -40,6 +40,8 @@ if (not $gcrypt_libpath and $] >= 5.006001) {
 # calculations for HMAC of some members of the SHA-2 family (including
 # SHA-512)
 check_lib_or_exit(
+    incpath => $gcrypt_incpath,
+    header => ['gcrypt.h'],
     function => 'if (gcry_check_version("1.3.0")) return 0; else return 1;',
     lib => [qw(gcrypt)],
     libpath => $gcrypt_libpath
